sklearn/naive_bayes.py

```diff
@@ -115,6 +115,16 @@ class GaussianNB(BaseNB):
 
     Read more in the :ref:`User Guide <gaussian_naive_bayes>`.
 
+    Parameters
+    ----------
+    fit_prior : boolean
+        Whether to learn class prior probabilities or not.
+        If false, a uniform prior will be used.
+
+    class_prior : array-like, size=(n_classes,)
+        Prior probabilities of the classes. If specified the priors are not
+        adjusted according to the data.
```

Comment:
Actually, you are still able to do this class fitting manually by giving the weights when calling the constructor, therefore you do not loose the feature, isn't it?